### PR TITLE
Edited DGUG, IncreaseCommand

### PR DESCRIPTION
**Pull request by Bisceto**
Edited DG:
- Added in introduction section and user stories

Edited UG:
- Added in inc and dec feature

Edited IncreaseCommand:
- Edited displayed success message
- Edited testcases to fit new success message

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1,8 +1,6 @@
 ---
-layout: page
-title: User Guide
+USER GUIDE FOR WIFE
 ---
-
 WIFE is always right. Our product Well Informed Fridge Environment &lt;WIFE/&gt; helps users to manage
```
Comment (ScorpiusSigma) [inline docs/UserGuide.md:4]: Do we want to change this out to Enhance life with WIFE?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -284,11 +317,19 @@ _Details coming soon ..._
 
 ## FAQ
 
-**Q**: *placeholder* <br/>
-**A**: *placeholder*
+**Q**: Can I use this application with other people? <br/>
+**A**: As of now, WIFE does not support concurrent users. If you would like to share your WIFE food list with another
+user, install WIFE on their computer and overwrite their data file with the data file created by WIFE in your computer.
```
Comment (ScorpiusSigma) [inline docs/UserGuide.md:322]: I think this is easily addressed by tags. You can use tags to identify user's items. To show all user item, we can use `listbytag n/wesley` to show all items with wesley tag

**Review by ScorpiusSigma — COMMENTED** [review]
As commented

File: docs/UserGuide.md (hunk 1 above)
Comment (Bisceto) [reply docs/UserGuide.md:4]: Will rectify after merging.

**Pull request merged**